webapp/src/Entity/ContestProblem.php

```diff
@@ -75,7 +76,7 @@ class ContestProblem
         options: ['comment' => 'Whether to do lazy evaluation for this problem; if set this overrides the global configuration setting', 'unsigned' => true]
     )]
     #[Serializer\Exclude]
-    private ?int $lazyEvalResults = null;
+    private int $lazyEvalResults = 0;
```

Comment:
We need a migration that:

* Updates existing `null` entries to `0`
* Makes the field non nullable

Reply:
Done.